### ISSUE TITLE
Uncaught Exception Error: Call to a member function xpath() on null in /var/www/html/glpi/src/RichText/UserMention.php at line 207

### ISSUE THREAD

**Issue by SilvaFernando** [glpi-project/glpi#17731]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.16

### Bug description

Hi Team.

I have this error when my mail collector tries to import emails:
Uncaught Exception Error: Call to a member function xpath() on null in /var/www/html/glpi/src/RichText/UserMention.php at line 207

I've checked this issue https://github.com/glpi-project/glpi/issues/16694 and mbstring is right.

### Relevant log output

```shell
Uncaught Exception Error: Call to a member function xpath() on null in /var/www/html/glpi/src/RichText/UserMention.php at line 207
```

**Comment by cconard96** [comment]
This should have been fixed in GLPI 10.0.11 by #15962. The fix still seems to be in place in 10.0.16. Although, the line number in your error message doesn't correspond to one that actually calls the xpath function in 10.0.16. Instead it matches the file from version 10.0.10 (and maybe some earlier versions).

**Comment by cconard96** [comment]
Please verify your GLPI is upgraded properly to 10.0.16 and then test again.

**Comment by trasher** [comment]
As @cconard96  said, this is exactly the issue that was fixed in #15962, with the exact same line, that is not the one in GLPI 10.0.16. Double check files in your instance.
